lib/crypto/tests.py

```diff
@@ -16,6 +15,7 @@
 from lib.crypto.receipt import crack, sign, SigningError
 from mkt.site.storage_utils import copy_to_storage
 from mkt.site.fixtures import fixture
+from mkt.site.storage_utils import private_storage as storage
```

Comment:
We should avoid `import <something> as <somethingelse>` if we can IMHO, it gets confusing. Especially so in this case.